src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/KeyTransRecipientInfoTests.cs

```diff
@@ -123,6 +123,36 @@ public static void TestKeyTransRecipientIdValue_Ski_RoundTrip()
         }
 
         [Fact]
+        [OuterLoop(/* Leaks key on disk if interrupted */)]
+        [ActiveIssue(3334, PlatformID.AnyUnix)]
+        public static void TestKeyTransRecipientIdValue_ExplicitSki_RoundTrip()
```

Comment:
You are using SKID in some places, and SKI in others.  Is there a semantic difference to your usage?